README.md

```diff
@@ -16,19 +16,29 @@ We’re working to update the [Scratch website](https://scratch.mit.edu) to use
 We’re currently building Scratch using [React](https://facebook.github.io/react/) and [SCSS](http://sass-lang.com/documentation/file.SASS_REFERENCE.html). Here are some resources to help you get acquainted with how we’re working on the Scratch codebase:
 
 * [Style Guide](https://github.com/LLK/scratch-www/wiki/Style-Guide)
-* [Testing Guide](https://github.com/LLK/scratch-www/wiki/Testing-Guide)
+* [Testing Guide](https://github.com/LLK/scratch-www/wiki/Testing-Guide-for-Bugfixes)
 * [Localization Guide](https://github.com/LLK/scratch-www/wiki/Localization-Guide)
 * [Map of the repository](https://github.com/LLK/scratch-www/wiki/Repo-Map)
 
 
 ### Before Getting Started
 * Make sure you have node (v4.2 or higher) and npm [installed](https://docs.npmjs.com/getting-started/installing-node)
+We use npm (Node Package Manager) to maintain and update packages required to build the site.  
+
+### Update Packages
+When making changes it's important to make sure that all of the node packages are up to date. Some changes in the code are needed because of package updates, so make sure you do this. You can update the packages by running this command:
```

Comment:
I think this is a little unclear, how about:

> It's important to make sure that all of the dependencies are up to date because the scratch-www code only works with specific versions of the dependencies. You can update the packages by running this command: